js/views/userPageVw.js

```diff
@@ -483,10 +483,9 @@ UserPageVw = pageVw.extend({
   },
 
   setState: function(state, hash, options) {
-    var currentAddress,
-        addressState,
+    var addressState,
```

Comment:
very minor... won't hold up the merge for this, but the common convention I've seen is that any vars that aren't assigned something at declaration are below vars that do assign something. So, it would be a matter of moving currentHandle above addressState.